src/main/java/com/frcteam3255/components/swerve/SN_SwerveModule.java

```diff
@@ -221,13 +220,10 @@ public void setModuleState(SwerveModuleState desiredState, boolean isOpenLoop) {
 		if (isOpenLoop) {
 			// Setting the motor to PercentOutput uses a percent of the motors max output.
 			// So, the requested speed divided by it's max speed.
-			double percentOutput = state.speedMetersPerSecond / maxModuleSpeedMeters;
-			driveMotor.set(ControlMode.PercentOutput, percentOutput);
-
+			driveMotor.setControl(new VoltageOut(0).withOutput(state.speedMetersPerSecond / maxModuleSpeedMeters));
```

Comment:
I believe VoltageOutput.withOutput is no longer a -1 to 1 scale. Documentation makes it look -12 to 12

https://pro.docs.ctr-electronics.com/en/latest/docs/migration/migration-guide/control-requests-guide.html#using-control-requests

Reply:
I'm just going off of this https://v6.docs.ctr-electronics.com/en/stable/docs/migration/migration-guide/control-requests-guide.html#using-control-requests